samples/django.md

```diff
@@ -137,9 +138,11 @@ In this step, you create a Django starter project by building the image from the
    If you are running Docker on Linux, the files `django-admin` created are
    owned by root. This happens because the container runs as the root user.
    Change the ownership of the new files.
+   
+   Do not change the permission of the data folder where Postgres has its file, otherwise Postgres will not be able to start due to permission issues.
 
    ```console
-   $ sudo chown -R $USER:$USER .
+   $ sudo chown -R $USER:$USER composeexample manage.py
```

Comment:
I would use `sudo chown -R $USER:$USER compose-example manage.py` for better reading.